backend/src/error.rs

```diff
@@ -16,12 +16,15 @@ use utoipa::ToSchema;
 pub enum ErrorReference {
     EntryNumberNotSupported,
     EntryNotFound,
-    PollingStationAlreadyFinalized,
+    PollingStationAlreadyFinalised,
+    PollingStationFirstEntryAlreadyFinished,
+    PollingStationFirstEntryNotFinished,
```

Comment:
Should we use `Finalised` instead of `Finished` here?

Reply:
Yeah, I agree that's probably more clear. Fixed in [deeaf03](https://github.com/kiesraad/abacus/pull/612/commits/deeaf0313f9e9d9c5e2889df3384e603a0c90068)